src/main/java/com/nearvanilla/deathdb/libs/Serialization.kt

```diff
@@ -0,0 +1,43 @@
+/* Licensed under GNU General Public License v3.0 */
+package com.nearvanilla.deathdb.libs
+
+import com.nearvanilla.deathdb.DeathDB
+import org.bukkit.inventory.ItemStack
+import org.json.simple.JSONArray
+import org.json.simple.JSONObject
+import org.json.simple.parser.JSONParser
+import java.util.*
+
+class Serialization {
+    companion object {
+        fun Serialize(inv: Array<ItemStack?>): String { // TODO Account for Armor and Shield slots.
```

Comment:
Old TODO? Not valid anymore, right? At least in this place.

Reply:
Probably left it in there by mistake, as from memory, this TODO was rectified. Will remove ASAP.

Reply:
TODO Removed.